scripts/bytecodecompare/prepare_report.py

```diff
@@ -54,15 +63,23 @@ def format_report(self) -> str:
         return report
 
 
-def load_source(path: Union[Path, str]) -> str:
+def load_source(path: Union[Path, str], smt_use: SMTUse) -> str:
     # NOTE: newline='' disables newline conversion.
     # We want the file exactly as is because changing even a single byte in the source affects metadata.
     with open(path, mode='r', encoding='utf8', newline='') as source_file:
         file_content = source_file.read()
 
+    if smt_use == SMTUse.STRIP_PRAGMAS:
+        return file_content.replace('pragma experimental SMTChecker;', '')
```

Comment:
```suggestion
        return file_content.replace('pragma experimental SMTChecker;', '', 1)
```
So that it'll be consistent with javascript `.replace`

Reply:
Done.